code/index.html

```diff
@@ -3,20 +3,79 @@
+      </article>
     </section>
 
-    <section class="">
+    <section class="other-news">
       <!-- The other news  -->
+      <article class="apples-article">
+        <img class="other-news-image" src="./image/apples-fanny-gustafsson.jpg" alt="four apples lying on the ground" />
+        <p class="other-news-text">An apple a day...</p>
+      </article>
+      <article class="meadow-article">
+        <img class="other-news-image" src="./image/meadow-anne-nygard.jpg" alt="a meadow in late summer" />
+        <p class="other-news-text">Lorem ipsum dolor...</p>
+      </article>
+      <article class="flower-article">
+        <img class="other-news-image" src="./image/flower-elisa-h.jpg" alt="closup of a white flower in natural habitat" />
+        <p class="other-news-text">Lorem ipsum dolor...</p>
+      </article>
+      <article class="fire-article">
+        <img class="other-news-image" src="./image/fire-isi-parente.jpg" alt="bonfire in a dark night" />
+        <p class="other-news-text">Lorem ipsum dolor...</p>
+      </article>
+      <article class="lilac-flowers-article">
+        <img class="other-news-image" src="./image/lilacflower-may-marketing.jpg" alt="closeup of lilac flowers in natural habitat" />
+        <p class="other-news-text">Lorem ipsum dolor...</p>
+      </article>
+      <article class="blackberry-article">
+        <img class="other-news-image" src="./image/Raspberry-amanda-hortiz.jpg" alt="closup of a twig with blackberries" />
+        <p class="other-news-text">Lorem ipsum dolor...</p>
+      </article>
+      <article class="mushrooms-article">
+        <img class="other-news-image" src="./image/mushrooms-andrew-ridley.jpg" alt="different sorts of mushrooms lying next to each other" />
+        <p class="other-news-text">Tasty treasures...</p>
+      </article>
     </section>
+
+    <section class="contact">
+      <div class="sm-icons" id="contact-section">
+        <a href="https://github.com/nehrwein" target="_blank"><i class="fab fa-github"></i></a>
+        <a href="https://codepen.io/BumBumBurka" target="_blank"><i class="fab fa-codepen"></i></a>
+        <a href="https://www.linkedin.com/in/birgit-nehrwein-9b4656194/" target="_blank"><i class="fab fa-linkedin"></i></a>
+        <a href="https://www.facebook.com/birgit.nehrwein.7/" target="_blank"><i class="fab fa-facebook"></i></a> 
+      </div>
+      <p class="contact-paragraph">Get in contact! :)</p>
+    </section>
+
+    <footer>
+      <p class="footer-text" id="about-footer">&copy 2021 BumBumBurka</p>
```

Comment:
There are both class and id in these paragraph, and I saw that you used the same syntax in other divs and sections. You can style with both, but would it be better to style only with classes in these case? Because id is very special and later on we are going to use them in JavaScript.

Reply:
Good point!

Reply:
OMG, now I know, why I need the IDs. They are serving as internal links on the site. 
So I have for example this About-Link in the navbar
`<a class="nav-link" id="about" href="#about-footer">About</a> `
that is jumping to the footer and therefore I need the ID 
`<p class="footer-text" id="about-footer">&copy 2021 BumBumBurka</p>`